systems/data-plan/api-gateway/pkg/rest/api.go

```diff
@@ -1,9 +1,11 @@
 package rest
 
+import "github.com/ukama/ukama/systems/common/uuid"
+
 type AddPackageRequest struct {
 	Name        string `json:"name" validation:"required"`
 	Duration    uint64 `json:"duration" `
-	OrgId       uint64 `json:"org_id" validation:"required"`
+	OrgID       uuid.UUID `json:"org_id" validation:"required"`
```

Comment:
OrgId here is UUID tyoe but in GetPackageByOrgRequest is uint64 ?
Also why not have packageId also as UUID. this info  is also shared across the systems.

Reply:
It has been changed to uuid all over.